backend/handlers/handlers.go

```diff
@@ -763,6 +764,59 @@ func (handlers *Handlers) getAccountsBalance(*http.Request) (interface{}, error)
 	return totalAmount, nil
 }
 
+// getCoinsTotalBalance returns the total balances grouped by coins.
+func (handlers *Handlers) getCoinsTotalBalance(_ *http.Request) (interface{}, error) {
+	totalPerCoin := make(map[coin.Code]*big.Int)
+	conversionsPerCoin := make(map[coin.Code]map[string]string)
+
+	totalAmount := make(map[coin.Code]coin.FormattedAmount)
```

Comment:
not a rebase error but this should be `coinpkg.FormattedAmount` - mixing imports.

Reply:
All the other FormattedAmount occurrences in the file are `coin.FormattedAmount`, I just thought to keep the same. Anyway I guess we should remove all the `coin.Something` calls and substitute them with `coinPkg.` to remove the double import.